go.mod

```diff
@@ -34,17 +34,14 @@ require (
 	github.com/rivo/uniseg v0.2.0 // indirect
 	github.com/spf13/cobra v1.2.0
 	github.com/submariner-io/admiral v0.10.0-rc0
-	github.com/submariner-io/cloud-prepare v0.10.0-m2
+	github.com/submariner-io/cloud-prepare v0.10.0-rc0
 	github.com/submariner-io/lighthouse v0.10.0-m2.0.20210618122405-aef0fb374a53
```

Comment:
Why isn't lighthouse at v0.10.0-m2 as well?

Reply:
Probably the bot script ignored the error with lighthouse v0.10.0-rc0 ^ @mkolesnik ?

Reply:
I suggest you run `go get github.com/submariner-io/lighthouse@v0.10.0-rc0 && go mod vendor && go mod tidy` and update this patch manually.